### PR TITLE
Hook up with Appveyor

### PR DESCRIPTION
**Pull request by Chocobo1**
[Appveyor](https://www.appveyor.com/) provides building & testing on windows platform.

Also I add the badges to `README.md`.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ os: Visual Studio 2015
 build_script:
   - CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\vcvars32.bat"
   - cd "%APPVEYOR_BUILD_FOLDER%\src"
-  - nmake -f Makefile.w32
-  - gifsicle --version
+  - nmake -f Makefile.w32 gifsicle.exe gifdiff.exe
+  - gifsicle.exe --version
+  - gifdiff.exe --version
```
Comment (h3xx) [inline .appveyor.yml]: This hunk is outside the scope of adding Travis CI badges to the readme.

Comment (Chocobo1) [inline .appveyor.yml]: Oops, fixed now.
Thanks for the review :)

**Comment by kohler** [comment]
Not sure I'll keep the badges forever, but why not for now. Thanks for the commit.

**Comment by kohler** [comment]
Appveyor badge appears broken.

**Comment by Chocobo1** [comment]
>Appveyor badge appears broken.

You'll have to sign up at Appveyor and add this project.
see: https://www.appveyor.com/docs/

**Comment by kohler** [comment]
Meh.

**Pull request merged**